x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.tsx

```diff
@@ -65,24 +65,36 @@ const FlyoutRiskSummaryComponent = <T extends EntityType>({
   isLinkEnabled,
   isPreviewMode,
 }: RiskSummaryProps<T>) => {
-  const { telemetry } = useKibana().services;
+  const { telemetry, spaces } = useKibana().services;
   const { data } = riskScoreData;
   const riskData = data && data.length > 0 ? data[0] : undefined;
   const entityData = getEntityData<T>(entityType, riskData);
   const { euiTheme } = useEuiTheme();
+  const [spaceId, setSpaceId] = useState('default');
```

Comment:
there is a `useSpaceId` hook at `x-pack/solutions/security/plugins/security_solution/public/common/hooks/use_space_id.ts` we can use here 👍